jpa/eclipselink/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<groupId>org.springframework.data.examples</groupId>
+		<artifactId>spring-data-jpa-examples</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<repositories>
+		<repository>
+			<id>com.ethlo.eclipselink.tools</id>
+			<url>http://ethlo.com/maven</url>
+		</repository>
+	</repositories>
+
+	<pluginRepositories>
+		<pluginRepository>
+			<id>com.ethlo.eclipselink.tools</id>
+			<url>http://ethlo.com/maven</url>
+		</pluginRepository>
+	</pluginRepositories>
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>example.springdata.jpa</groupId>
+	<artifactId>eclipselink</artifactId>
```

Comment:
Please change to `spring-data-jpa-eclipselink`to be consistent with the other modules.